src/pages/addCard.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components';
+import { CardRegisterForm } from '../components/CardRegisterForm';
+import { Header, BackButton } from '../components/common';
+import { CardInfoProvider } from '../contexts/cardInfo';
+import { CardPreview } from '../components/common/CardPreview';
+
+export function AddCard() {
+  return (
+    <AddCardContainer>
+      <Header title='카드 추가' render={() => BackButton()} />
```

Comment:
불필요하게 익명함수를 사용할 필요가 없을거 같아요~

```suggestion
<Header title="카드 추가" render={BackButton} />
```